ansible/roles/content-snapshot-indexer/templates/scripts/DruidContentIndexer.j2

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+set -e
```

Comment:
You should use ```set -eo pipefail``` to stop the script in case of issues and also return correct code to Jenkins